docs/proposals/InputColor/README.md

```diff
@@ -0,0 +1,64 @@
+# InputColor
+
+The `<InputColor>` component will ... _{Add a brief description of the
+component}_
+
+## Design Patterns
+
+\_{The goal of the `InputColor` component is to allow a color be chosen to
+reflect the brand of an SP
+
+An SP can either pick a color from the color pick box or enter in the hex value
+and both the color box and the hew value should reflect eachother }\_
+
+## Wireframe
+
+![mockup](https://user-images.githubusercontent.com/51097786/62891872-1b79d980-bd04-11e9-9c5e-a66955519674.png)
+
+## Interface
+
+```jsx
```

Comment:
For the interface here consider less of the implementation side and more on what you will be providing it and how it will make use of that.

https://github.com/GetJobber/atlantis/pull/66/files

Reply:
after our talk yesterday I believe this is what you are looking for.